sdcm/report_templates/results_latency_during_ops_short.html

```diff
@@ -78,21 +78,24 @@ <h2>{{ operation }}</h2>
                                 <th>Latency {{ lat_type }}</th>
                                 <th>Steady State {{lat_type}}</th>
                             {% endfor %}
+                            <th> Duration (sec) </th>
 
                             <th>Commit id, date</th>
                         </tr>
                         {% for cycle in results["cycles"] %}
                             {% set row_span = cycle["hdr_summary"]|length %}
+                            {% set cycle_index = loop.index %}
                             {% for workload in cycle["hdr_summary"] %}
                                 <tr>
                                     {% if loop.first %}
-                                            <td rowspan="{{ row_span }}"> Cycle #{{ loop.index }} </td>
+                                            <td rowspan="{{ row_span }}"> Cycle #{{ cycle_index }} </td>
```

Comment:
this is the fix for the index showing always number 1 cycle, right?

Reply:
yes